### PR TITLE
[DOCS-5719]: Infrastructure Vulnerability Management

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do?
Adds docs for Infrastructure Vulnerability Management

Motivation
[DOCS-5719](https://datadoghq.atlassian.net/browse/DOCS-5719)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5719]: https://datadoghq.atlassian.net/browse/DOCS-5719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] uses the Datadog Agent to monitor your container images and identify known vulnerabilities for you to prioritize and remediate. The Agent scans running container images on hosts and creates a software bill of materials (SBOM). The SBOM is then scanned for vulnerabilities.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:21]: Datadog [Infrastructure Vulnerability Management][1] leverages deep infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, and its ownership, across your organization.

This capability is currently in Private Beta. It offers vulnerabilities in the Container Images on your hosts. If you are enrolled in the Private Beta, you will see additional capabilities light up with existing configuration.

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Review the health of your container images
+
+View a list of running container images on the [Container Images][2] page. Identify container images with vulnerabilities and view additional details about the vulnerability on the details page.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:25]: View your container images on the [Container Images][2] page. Sort container images by number of active containers (column name "CONTAINERS") and see the count of vulnerabilities that exist in the container image (column name "VULNERABILITIES").

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+{{< img src="security/infrastructure_vulnerability/container_images.png" alt="Container Images overview page" width="100%">}}
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:27]: Can we use this new (attached) image here?

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:28]: View additional details about the vulnerability by clicking on the container image and reviewing the "Vulnerabilities" tab.

(Use the attached image here)

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+## Identify the top container-based vulnerabilities
+
+Prioritize vulnerabilities for remediation based on severity on the [Vulnerabilities][3] page. View additional information about each vulnerability on the details page, including number of impacted images, the date on which the vulnerability was last detected, a description of the vulnerability, and recommended remediation steps.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:31]: Prioritize vulnerabilities for remediation based on severity on the [Vulnerabilities][3] page in Cloud Security Management (link).

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+{{< img src="security/infrastructure_vulnerability/infra_vulnerability_explorer.png" alt="Infrastructure Vulnerability Management explorer page" width="100%">}}
+
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:34]: View additional information about each vulnerability by clicking on the vulnerability. This information includes information about affected container images, the date on which the vulnerability was last detected, a description of the vulnerability, and recommended remediation steps.

(Use attached screenshot)

File: content/en/security/infrastructure_vulnerability/setup.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+## Pre-requisites
+
+- [Cloud Security Management][1] (using [Cloud Workload Security][2] or [Cloud Security Posture Management][3]).
+- Datadog Agent 7.44.0 or higher.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/setup.md:20]: Datadog Agent 7.45.0 or higher.

File: content/en/security/infrastructure_vulnerability/setup.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+- [Helm Chart][5] 3.22.0 or higher.
+- 5GB or more of free disk space on the Datadog Agent host. 
+
+**Note**: In order for the Datadog Agent to scan for vulnerabilities, the free disk space must be equivalent to the size of your largest container image. If you have container images larger than 5GB, you would need to have additional equivalent free disk space on the host. 
+
+## Setup
+
+Add the following options to your `values.yaml` configuration file:
+
+**Note**: This enables the Container Image Metadata collection (allowing you to explore container images) and SBOM collection in your Datadog Agent, allowing you to scan libraries present in container images and detect vulnerabilities.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/setup.md:31]: **Note**: This enables the Container Image Metadata collection (allowing you to explore container images) and SBOM collection in your Datadog Agent (allowing you to scan libraries present in container images and detect vulnerabilities).

File: content/en/security/infrastructure_vulnerability/_index.md (hunk 3 above)
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:27]: Can we please use the screenshot where the view (Infrastructure --> Containers --> Container Images) is sorted (desc) by "CONTAINERS" column? This way, the hottest container images are surfaced on the top. They have the most vulnerabilities.

**Review by rajat-luthra-ddog — CHANGES_REQUESTED** [review]
Thanks @aliciascott! Review comments added.

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] provides infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, and its ownership, across your organization. Once you are enrolled in the private beta, the [Container Images][2] page and banner with additional capabilities are viewable.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:21]: Datadog [Infrastructure Vulnerability Management][1] leverages infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, its use and its ownership in your organization. Currently, the private beta offers visibility into the vulnerabilities in the [Container Images][2] on your hosts. More functionality will be added during the course of the private beta.

(Can you please replace the first screenshot to one where the banner is closed (top-right X)?)

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+Datadog [Infrastructure Vulnerability Management][1] provides infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, and its ownership, across your organization. Once you are enrolled in the private beta, the [Container Images][2] page and banner with additional capabilities are viewable. 
+
+{{< img src="security/infrastructure_vulnerability/banner.png" alt="Banner Page" width="100%">}}
+
+## Review the health of your container images
+
+Infrastructure Vulnerability Management offers vulnerabilities in the Container Images on your hosts.
+View your container images on the [Container Images][2] page. Sort images by the number of active containers (column name `CONTAINERS`), observe the number of vulnerabilities that exist in the container image (column name `VULNERABILITIES`).
+View additional details about the vulnerability by clicking on the container image and reviewing the "Vulnerabilities" tab.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:29]: View additional details about any vulnerability by clicking on the container image and reviewing the "Vulnerabilities" tab.

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+{{< img src="security/infrastructure_vulnerability/container_image.png" alt="Container Images overview page" width="100%">}}
+
+{{< img src="security/infrastructure_vulnerability/container_vulnerability.png" alt="Container Images overview page" width="100%">}}
+
+
+## Identify the top container-based vulnerabilities
+
+Prioritize vulnerabilities for remediation based on severity on the [Vulnerabilities][3] page in [Cloud Security Management][4].
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:38]: View a consolidated list of all vulnerabilities on the [Vulnerabilities][3] page in [Cloud Security Management][4]. Prioritize them for remediation based on severity.

File: content/en/security/infrastructure_vulnerability/setup.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+## Pre-requisites
+
+- [Cloud Security Management][1] (using [Cloud Workload Security][2] or [Cloud Security Posture Management][3]).
+- Datadog Agent 7.45.0 or higher.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/setup.md:20]: - Datadog Agent v7.45.0 or higher.

**Review by rajat-luthra-ddog — CHANGES_REQUESTED** [review]
Thanks Alicia!

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] leverages infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, and its ownership, across your organization. The private beta offers visibility into the vulnerabilities in the [Container Images][2] on your host.
+
+## Review the health of your container images
+
+Infrastructure Vulnerability Management offers vulnerabilities in the Container Images on your hosts.
+View your container images on the [Container Images][2] page. Sort images by the number of active containers (column name `CONTAINERS`), observe the number of vulnerabilities that exist in the container image (column name `VULNERABILITIES`).
```
Comment (estherk15) [inline content/en/security/infrastructure_vulnerability/_index.md:26]: I like breaking up features into bullet points, but it's not a requirement just a preference in formatting:
```suggestion
- View your container images on the [Container Images][2] page. 
- Sort images by the number of active containers (column name `CONTAINERS`).
- Observe the number of vulnerabilities that exist in the container image (column name `VULNERABILITIES`).
```

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+View additional details about any vulnerability by clicking on the container image and reviewing the "Vulnerabilities" tab.
+
+{{< img src="security/infrastructure_vulnerability/container_image.png" alt="Container Images overview page" width="100%">}}
```
Comment (estherk15) [inline content/en/security/infrastructure_vulnerability/_index.md:29]:
```suggestion
{{< img src="security/infrastructure_vulnerability/container_image.png" alt="Container Images overview page highlighting the vulnerabilities and container column sort feature" width="100%">}}
```

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+
+{{< img src="security/infrastructure_vulnerability/container_vulnerability.png" alt="Container Images overview page" width="100%">}}
```
Comment (estherk15) [inline content/en/security/infrastructure_vulnerability/_index.md:31]:
```suggestion
{{< img src="security/infrastructure_vulnerability/container_vulnerability.png" alt="Container Images overview page highlighting the vulnerabilities tab" width="100%">}}
```

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Identify the top container-based vulnerabilities
+
+View a consolidated list of all vulnerabilities on the [Vulnerabilities][3] page in [Cloud Security Management][4].
+Prioritize them for remediation based on severity.
+View additional information about each vulnerability by clicking on the vulnerability. This information includes information about affected container images, the date on which the vulnerability was last detected, a description of the vulnerability, and recommended remediation steps.
```
Comment (estherk15) [inline content/en/security/infrastructure_vulnerability/_index.md:38]:
```suggestion
View additional details by clicking on the vulnerability. This opens a panel which includes information about: 
- Affected container images.
- The date on which the vulnerability was last detected.
- A description of the vulnerability.
- Recommended remediation steps.
```

File: content/en/security/infrastructure_vulnerability/setup.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+  text: "Infrastructure Vulnerability Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+## Pre-requisites
```
Comment (estherk15) [inline content/en/security/infrastructure_vulnerability/setup.md:17]:
```suggestion
## Prerequisites
```

File: content/en/security/infrastructure_vulnerability/setup.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+        - chunk_size: 1
+```
+
+
+[1]: https://docs.datadoghq.com/security/cloud_security_management
+[2]: https://docs.datadoghq.com/security/cloud_workload_security/
+[3]: https://docs.datadoghq.com/security/cspm/
+[4]: https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/
+[5]: https://docs.datadoghq.com/containers/kubernetes/installation/?tab=helm
+[5]: https://docs.datadoghq.com/agent/
```
Comment (estherk15) [inline content/en/security/infrastructure_vulnerability/setup.md:64]:
```suggestion
```

**Review by estherk15 — CHANGES_REQUESTED** [review]
Hi Alicia, left some notes, let me know if you have any questions.

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] leverages infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, and its ownership, across your organization. The private beta offers visibility into the vulnerabilities in the [Container Images][2] on your host.
```
[the remark this answers is not in the document]
Comment (aliciascott) [reply content/en/security/infrastructure_vulnerability/_index.md:24]: Changed to:

"Datadog [Infrastructure Vulnerability Management][1] leverages infrastructure observability to detect, prioritize, and manage vulnerabilities in your organization. The private beta offers visibility into the vulnerabilities in the [Container Images][2] on your hosts."

**Review by estherk15 — APPROVED** [review]

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 20]
```diff
@@ -0,0 +1,63 @@
...
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">Infrastructure Vulnerability Management is in beta.</div>
+
+## Overview
+
+Datadog [Infrastructure Vulnerability Management][1] leverages infrastructure observability to detect, prioritize, and manage vulnerabilities in your organization. Infrastructure Vulnerability Management offers visibility into the vulnerabilities in the [Container Images][2] and hosts in your infrastructure.
```
Comment (rajat-luthra-ddog) [inline content/en/security/infrastructure_vulnerability/_index.md:20]: @aliciascott let's change this to: Datadog Infrastructure Vulnerability Management combines observability data with security insights to detect, prioritize and manage vulnerabilities in your organization. It offers visibility into the vulnerabilities in the [container images] [2] and [hosts] [We should link to this as well] in your infrastructure.

Comment (aliciascott) [reply content/en/security/infrastructure_vulnerability/_index.md]: Updated

**Comment by aliciascott** [comment]
@clamoriniere can I get your review on the Setup page? Particularly the Hosts setup section as I'm missing some information there on how to mount the host volume(s). Thanks!

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 21]
```diff
@@ -0,0 +1,64 @@
...
+
+## Review the health of your container images
+Infrastructure Vulnerability Management offers vulnerabilities in the Container Images on your hosts.
+
+- View your container images on the [container images][2] page. 
+- Observe the number of vulnerabilities that exist in the container images (column name `VULNERABILITIES`).
+
+{{< img src="security/infrastructure_vulnerability/container_image_2.png" alt="Images overview page highlighting the vulnerabilities and container column sort feature" width="100%">}}
+
+View additional details about any vulnerability by clicking on the container image and reviewing the **Vulnerabilities** tab:
```
Comment (drichards-87) [inline content/en/security/infrastructure_vulnerability/_index.md:30]:
```suggestion
View additional details about any vulnerability by clicking the container image and reviewing the **Vulnerabilities** tab:
```

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 22]
```diff
@@ -0,0 +1,64 @@
...
+
+{{< img src="security/infrastructure_vulnerability/container_vulnerability.png" alt="Container Images overview page highlighting the vulnerabilities tab" width="100%">}}
+
+## Identify impacted infrastructure
+
+- Click on any vulnerability to see more information. 
+- Click on "**View Vulnerability in CSM**" to view all related infrastructure impacted by this vulnerability:
```
Comment (drichards-87) [inline content/en/security/infrastructure_vulnerability/_index.md:37]:
```suggestion
- Click any vulnerability to see more information. 
- Click **View Vulnerability in CSM** to view all related infrastructure impacted by this vulnerability:
```

File: content/en/security/infrastructure_vulnerability/_index.md [hunk 23]
```diff
@@ -0,0 +1,64 @@
...
+- A description of the vulnerability.
+- The date on which the vulnerability was last detected.
+- Recommended remediation steps.
+- Affected container images and hosts.
+
+{{< img src="security/infrastructure_vulnerability/CSM-Infra-Vulnerability_3.png" alt="CSM Infrastructure Vulnerability Explorer view, details of a vulnerability" width="100%">}}
+
+## Review and remediate
+
+View a consolidated list of all vulnerabilities within the [Vulnerabilities][1] explorer in [Cloud Security Management][3]. Prioritize them for remediation using available facets, such as context-based **severity**, **resource type**, and **vulnerability source**. Additionally you can sort vulnerabilities by **library**, **registry image**, **host**, **environment**, and more.
```
Comment (drichards-87) [inline content/en/security/infrastructure_vulnerability/_index.md:52]:
```suggestion
View a consolidated list of all vulnerabilities within the [Vulnerabilities][1] explorer in [Cloud Security Management][3]. Prioritize them for remediation using available facets, such as context-based **severity**, **resource type**, and **vulnerability source**. Additionally, you can sort vulnerabilities by **library**, **registry image**, **host**, **environment**, and more.
```

**Review by drichards-87 — APPROVED** [review]
Looks good! I only had a couple of minor suggestions.

File: content/en/security/infrastructure_vulnerability/setup.md [hunk 24]
```diff
@@ -0,0 +1,129 @@
...
+{{< /site-region >}}
+
+<div class="alert alert-info">Infrastructure Vulnerability Management is in beta.</div>
+
+## Prerequisites
+
+- [Cloud Security Management][1] (using [Cloud Workload Security][2] or [Cloud Security Posture Management][3]).
+- [Datadog Agent][4] 7.46.0 or higher.
+- [containerd][5] v1.5.6 or higher.
+- [Helm Chart][6] v3.22.0 or higher.
```
Comment (clamoriniere) [inline content/en/security/infrastructure_vulnerability/setup.md:23]:
```suggestion
- [Helm Chart][6] v3.33.6 or higher.
```

**Pull request merged**